src/content/manifest/overview-manifest.mdx

```diff
@@ -0,0 +1,153 @@
+
+> Note: If you have external resources you'd like to configure and show within the Okteto development environment, you must configure those in this section.
+
+Your `deploy` section might look like this:
+
+```yaml
+deploy:
+    - name: Deploy PostgreSQL
+    command: helm upgrade --install postgresql postgresql/postgresql-11.6.21.tgz -f postgresql/values.yml --version 11.6.21
+    - name: Deploy Frontend
+    command: helm upgrade --install frontend frontend/chart --set image=${OKTETO_BUILD_FRONTEND_IMAGE}
+    - name: Deploy API
+    command: helm upgrade --install api api/chart --set image=${OKTETO_BUILD_API_IMAGE} --set load=${API_LOAD_DATA:-true}
+```
+
+In this section the manifest instructs Okteto to execute three commands: deploy a postgresql database, deploy the `frontend` image (using an environment variable), and deploy the `api` image (also using environment variables).
+
+These commands are executed within the development environment created by Okteto enabling you to create a production-like environment for developing your application.
+
+## Destroy
+
+The commands you define in this section will ensure any resources external to the Kubernetes cluster Okteto manages are destroyed when deleting the development environment. 
+
+> Note: The Okteto CLI command `okteto destroy` will automatically destroy any Kubernetes resources created by the `okteto deploy` command.
+
+Your `destroy` section might look like this:
+
+```yaml
+destroy:
+  image: okteto/pipeline-runner:1.0.0-sam
+  commands:
+    - name: destroy worker service
+      command: |
+        sam delete --no-prompts --stack-name "${OKTETO_NAMESPACE}-voting-texkhnclxd" --region us-east-1
+```
+
+In this example `destroy` section we are deleting the worker service running on the image container specified in the `image:` attribute. When the development environment is deleted, Okteto will execute this command ensuring orphaned resources don't remain active.
+
+## External
+
+This section contains a list of resources external to the Okteto cluster such as cloud resources or dashboards, but that are part of your development environment. Configuring your external resources enables you to build a more complete development experience in Okteto. 
+
+Your `external` section might look like this:
+
+```yaml
+external:
+  readme:
+    icon: okteto
+    notes: README.md
+    endpoints:
+    - name: Try it out!
+      url: https://github.com/okteto/voting-app-with-external-resources
+  lambda:
+    icon: aws
+    notes: docs/lambda.md
+    endpoints:
+    - name: function
+```
+
+This section provides only the metadata to represent and link to external resources in the Okteto UI and does not create any resources in a cloud provider. Creating or destroying the resources themselves is done in the [deploy](#deploy) and [destroy](#destroy) sections, respectively.
```

Comment:
Should we add a link to the `external` reference docs?

Reply:
```suggestion
This section provides only the metadata to represent and link to external resources in the Okteto UI and does not create any resources in a cloud provider. Creating or destroying the resources themselves is done in the [deploy](#deploy) and [destroy](#destroy) sections, respectively.

You can learn more about this section [here](reference/okteto-manifest.mdx#external-object-optional).
```